lib/http/request.rb

```diff
@@ -199,6 +199,10 @@ def inspect
       "#<#{self.class}/#{@version} #{verb.to_s.upcase} #{uri}>"
     end
 
+    def self.example
+      new(:verb => :get, :uri => "http://example.com")
+    end
```

Comment:
Would you mind moving this into the specs themselves so it isn't otherwise defined when the specs aren't running? You can still define it on `HTTP::Request`

Reply:
I'm ok with removing the example from the `Request` definition, but don't think it is worth re-opening the class in the specs. I think that would be more confusing to readers to see the class method invoked, but not defined where you would expect to find it.

I think the specs will be more readable to just repeat `new(:verb => :get, :uri => "http://example.com")` where needed. I can make that change if you agree.

Reply:
Sure, that sounds good